sdk/durabletask/azure-resourcemanager-durabletask/src/main/java/com/azure/resourcemanager/durabletask/models/PurgeableOrchestrationState.java

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) TypeSpec Code Generator.
+
+package com.azure.resourcemanager.durabletask.models;
+
+import com.azure.core.util.ExpandableStringEnum;
+import java.util.Collection;
+
+/**
+ * Purgeable orchestration state to be used in retention policies.
+ */
+public final class PurgeableOrchestrationState extends ExpandableStringEnum<PurgeableOrchestrationState> {
+    /**
+     * The orchestration is completed.
+     */
+    public static final PurgeableOrchestrationState COMPLETED = fromString("Completed");
+
+    /**
+     * The orchestration is failed.
+     */
+    public static final PurgeableOrchestrationState FAILED = fromString("Failed");
+
+    /**
+     * The orchestration is terminated.
+     */
+    public static final PurgeableOrchestrationState TERMINATED = fromString("Terminated");
+
+    /**
+     * The orchestration is terminated.
```

Comment:
I fixed this doc string in the Typespec / swagger now so it should generate correctly the next time. At least the constant name and value are correct, so we probably do not need to re-release this.